### PR TITLE
EES-2554 move legacy releases to edit publication page

### PR DESCRIPTION
**Pull request by amyb-hiveit**
Moves managing legacy release from it's own page to the bottom of the manage publication page.

The same permissions as currently are used to display the legacy releases section, so publication owners will not see it. https://dfedigital.atlassian.net/browse/EES-2553 covers fixing the permissions.

legacy-releases

### CODE REVIEW

File: src/explore-education-statistics-admin/src/pages/legacy-releases/LegacyReleasesPageContainer.tsx [hunk 1]
```diff
@@ -3,17 +3,12 @@ import { PublicationContextProvider } from '@admin/contexts/PublicationContext';
...
 import { PublicationRouteParams } from '@admin/routes/routes';
 import React from 'react';
 import { RouteComponentProps, Switch } from 'react-router';
 
-const routes = [
-  legacyReleasesIndexRoute,
-  legacyReleaseCreateRoute,
-  legacyReleaseEditRoute,
-];
+const routes = [legacyReleaseCreateRoute, legacyReleaseEditRoute];
```
Comment (duncan-at-hiveit) [inline src/explore-education-statistics-admin/src/pages/legacy-releases/LegacyReleasesPageContainer.tsx:11]: Do we actually still need this Component if we no longer have a stand-alone Legacy Releases set of pages?

File: src/explore-education-statistics-admin/src/pages/legacy-releases/components/LegacyReleasesTable.tsx [hunk 2]
```diff
@@ -1,83 +1,62 @@
...
+import publicationService, {
+  BasicPublicationDetails,
+} from '@admin/services/publicationService';
 import Button from '@common/components/Button';
 import ButtonGroup from '@common/components/ButtonGroup';
-import LoadingSpinner from '@common/components/LoadingSpinner';
 import ModalConfirm from '@common/components/ModalConfirm';
 import useToggle from '@common/hooks/useToggle';
 import reorder from '@common/utils/reorder';
+import styles from '@admin/pages/legacy-releases/components/LegacyReleasesTable.module.scss';
```
Comment (duncan-at-hiveit) [inline src/explore-education-statistics-admin/src/pages/legacy-releases/components/LegacyReleasesTable.tsx:18]: Does it make sense to rehome this css module under a different folder now?

`pages/legacy-releases` suggests that it still belongs to a "Legacy Releases" page.

File: src/explore-education-statistics-admin/src/pages/legacy-releases/components/LegacyReleasesTable.tsx [hunk 3]
```diff
@@ -1,83 +1,62 @@
...
+const LegacyReleasesTable = ({ publication }: Props) => {
   const [isReordering, toggleReordering] = useToggle(false);
   const [deleteLegacyRelease, setDeleteLegacyRelease] = useState<
     LegacyRelease
   >();
 
-  if (!publication) {
-    return <LoadingSpinner loading={isLoading} />;
-  }
+  const [legacyReleases, setLegacyReleases] = useState<LegacyRelease[]>(
```
Comment (duncan-at-hiveit) [inline src/explore-education-statistics-admin/src/pages/legacy-releases/components/LegacyReleasesTable.tsx:33]: I *think* that you don't need the `<LegacyRelease[]>` part here, as useState can infer its type based upon you passing `publication.legacyReleases` into it.

File: src/explore-education-statistics-admin/src/pages/legacy-releases/LegacyReleasesPageContainer.tsx (hunk 1 above)
Comment (amyb-hiveit) [reply src/explore-education-statistics-admin/src/pages/legacy-releases/LegacyReleasesPageContainer.tsx:11]: The create and edit pages are still stand-alone, this is needed to wrap them in the context.

File: src/explore-education-statistics-admin/src/pages/legacy-releases/components/LegacyReleasesTable.tsx (hunk 2 above)
Comment (amyb-hiveit) [reply src/explore-education-statistics-admin/src/pages/legacy-releases/components/LegacyReleasesTable.tsx:18]: It should be in the same folder as the Table component. Though thinking about it, maybe that should be under publication instead as it's used on the edit publication page.

File: src/explore-education-statistics-admin/src/pages/legacy-releases/components/LegacyReleasesTable.tsx (hunk 3 above)
Comment (amyb-hiveit) [reply src/explore-education-statistics-admin/src/pages/legacy-releases/components/LegacyReleasesTable.tsx:33]: You are correct :)

**Review by duncan-at-hiveit — APPROVED** [review]

**Pull request merged**